port/win32/omrsl.c

```diff
@@ -478,5 +480,62 @@ findError(int32_t errorCode)
 uintptr_t
 omrsl_get_libraries(struct OMRPortLibrary *portLibrary, OMRLibraryInfoCallback callback, void *userData)
 {
-	return OMRPORT_ERROR_NOT_SUPPORTED_ON_THIS_PLATFORM;
+	HANDLE hProcess = GetCurrentProcess();
+	HMODULE *modules = NULL;
+	DWORD needed = 0;
+	uint32_t numModules = 0;
+	uint32_t i = 0;
+	uintptr_t result = 0;
+	if ((0 == EnumProcessModules(hProcess, NULL, 0, &needed)) || (0 == needed)) {
+		int32_t portableError = portLibrary->error_last_error_number(portLibrary);
+		portLibrary->error_set_last_error_with_message(
+				portLibrary,
+				portableError,
+				"Failed to get module size using EnumProcessModules.");
+		return (uintptr_t)(intptr_t)portableError;
```

Comment:
`error_last_error_number()` won't return anything useful for failed system calls; please just use `OMRPORT_ERROR_OPFAILED`:
```c
		portLibrary->error_set_last_error_with_message(
				portLibrary,
				OMRPORT_ERROR_OPFAILED,
				"Failed to get module size using EnumProcessModules.");
		return (uintptr_t)(intptr_t)OMRPORT_ERROR_OPFAILED;
```
Similar comment for lines 512-518.